stack.yaml

```diff
@@ -41,6 +41,11 @@ packages:
 # (e.g., acme-missiles-0.3)
 # extra-deps: []
 
+extra-deps:
+# We need githash 0.1.3.3 in order to have git-worktree(1) support. It
```

Comment:
Happy to upgrade the resolver as soon as the fixed version makes it into an LTS.